src/Cody.Core/DocumentSync/DocumentSyncCallback.cs

```diff
@@ -124,12 +124,10 @@ public void OnOpened(string fullPath, string content, DocumentRange visibleRange
                 };
             }
 
-            var docState = new ProtocolTextDocument
+            Range selRange = null;
```

Comment:
Let's put in addition whole OnOpened() code block inside try/catch with error logging `logger.Error()`